data/human/culture conversations.txt

```diff
@@ -163,6 +191,33 @@ mission "Class Trip to Port"
 
 
 
+mission "Blind Landing"
+	minor
+	landing
+	source
+		government "Free Worlds"
+		attributes "farming"
+		not attributes "station" "frontier"
+	to offer
+		random < 1
+		has "Trash Fire Crops:declined"
+	on offer
+		conversation
+			`Smog fills your windscreen as you make your descent, trying and failing to see anything more than a few metres in front of you. The autopilot systems show solid green across the full range of indicators though, and traffic control has confirmed a lock.`
+			choice
+				`	(Land on autopilot.)`
+				`	(Return to orbit.)`
+					goto orbit
+			`	You let the craft perform the landing, staring blankly at the haze until it clears.`
+			`	On the ground, you see the cause, fields upon fields of plantations burning. You see people arguing with firefighters, trying to stop them from putting out the flames as planes fly overhead, dropping water and retardants wherever necessary.`
+			`	You recall seeing something like this years ago, of farmers burning their crops to make the land fertile, and how bad the air was for months.`
```

Comment:
But what if the player gets offered this mission right after the last one?

Reply:
<1% chance?

Reply:
A 1% offer chance still leave room for this mission being offered close enough that "years ago" isn't an accurate descriptor. If you want to use language like this I'd say better safe than sorry and use an event with a 2 year delay. (If you want to get fancy with it you could instead do something like have `"bumper crop year" = "year"` in the `on offer` of the other mission and `"year" > "bumper crop year" + 2` in the `to offer` of this mission.)